package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "bl": "^1.1.2",
     "from2-string": "^1.1.0",
-    "static-module": "^1.3.0",
+    "static-module": "github:ahdinosaur/static-module#dynamic",
```

Comment:
Let's not merge this patch until this lands in `static-module` as github urls don't resolve from cache